hello/lib/hello_web/router.ex

```diff
@@ -18,7 +19,10 @@ defmodule HelloWeb.Router do
     pipe_through :browser
 
     get "/", PageController, :index
+
+    resources "/users", UserController
```

Comment:
@Vretamoza what is the main difference between resources and get/post?

Reply:
When we use the resources verb, it creates all the standard verbs, paths and controller actions for that request, when we use the get/post verb, we just create that individual request to a single path and invoke a single controller action.